pages/releases.md

```diff
@@ -0,0 +1,105 @@
+
+For example, for the feature release 1.3 we had several incremental releases including 1.3.10, 1.3.20, and 1.3.70.
+For 1.3.70, we had 2 bug fix releases – 1.3.71 and 1.3.72.
+
+For each incremental and feature release, we also ship several [preview versions](eap/index.html) that allow you to try 
+new features before they are released (Early Access Preview).
+
+Learn more about [types of Kotlin releases and their compatiblity](docs/reference/evolution/kotlin-evolution.html#feature-releases-and-incremental-releases). 
+
+## Updating to a new release
+
+IntelliJ IDEA and Android Studio suggest updating to a new release once it is out. When you accept the suggestion,
+it automatically updates the Kotlin plugin to the new version. You can check the Kotlin version in **Tools** \| **Kotlin** 
+\| **Configure Plugin Updates**.
+
+If you have projects created with earlier Kotlin versions, change the Kotlin version in your projects and update kotlinx
+libraries if necessary – check [compatible versions](#release-details).
+
+If you are migrating to the new feature release, Kotlin plugin's migration tools will help you with the migration.
+
+## Release details
+
+The following table lists details of latest Kotlin releases.
+
+<table>
+    <tr>
+        <th>Build info</th>
+        <th>Build highlights</th>
+        <th>Recommended kotlinx library versions</th>
+    </tr>
+    <tr>
+        <td><strong>1.4.0</strong>
+            <p> Released: <strong>August 17, 2020</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.4.0" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+            <p>A feature release with many features and improvements that mostly focus on quality and performance.</p>
+            <p>Learn more in:</p>
+            <ul>
+                <li><a href="http://blog.jetbrains.com/kotlin/2020/08/kotlin-1-4-released-with-a-focus-on-quality-and-performance/" target="_blank">Release blog post</a></li>
+                <li><a href="docs/reference/whatsnew14.html" target="_blank">What's new in Kotlin 1.4.0</a></li>
+                <li><a href="docs/reference/compatibility-guide-14.html" target="_blank">Compatibility Guide</a></li>
+                <li><a href="docs/reference/whatsnew14.html#migrating-to-kotlin-14" target="_blank">Migrating to Kotlin 1.4.0</a></li>
+            </ul>
+         </td>
+        <td>
+            <ul>
+                <li><a href="https://github.com/Kotlin/kotlinx.serialization" target="_blank"><strong>kotlinx.serialization</strong>
+                </a> version: <a href="https://bintray.com/kotlin/kotlinx/kotlinx.serialization.runtime/1.0.0-rc" target="_blank">1.0.0-rc</a></li>
+                <li><a href="https://github.com/Kotlin/kotlinx.coroutines" target="_blank"><strong>kotlinx.coroutines</strong></a>
+                version: <a href="https://bintray.com/kotlin/kotlinx/kotlinx.coroutines/1.3.9" target="_blank">1.3.9
```

Comment:
It is better to give links to the https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.3.8 pagesbecause there is a changelog. Same for all other libraries where there is corresponding releases pages:
https://github.com/Kotlin/kotlinx.atomicfu/releases
https://github.com/Kotlin/kotlinx.serialization/blob/master/CHANGELOG.md#0200--2020-03-04 for serialization
https://github.com/ktorio/ktor/releases
https://github.com/Kotlin/kotlinx.html/releases
(for node-js there is no fancy page, so bintray it is)

Reply:
Maybe it's better to give links to exact versions instead of `/releases` because with `/releases` they get inconsistent once a newer version is out?

Reply:
Yes, I mean that